Live-Snap/Live-Snap/Controllers/SelectPhotoViewController.swift

```diff
@@ -70,15 +73,46 @@ class SelectPhotoViewController: UIViewController, UICollectionViewDataSource, U
         progressView = UIView()
         activityIndicator = UIActivityIndicatorView()
         
+        deniedLibraryPermissionLabel = UILabel(frame: CGRect(x: 0, y: 0, width: view.frame.width * 0.75, height: 100))
+        deniedLibraryPermissionLabel.center = view.center
+        deniedLibraryPermissionLabel.textColor = UIColor.snapWhite
+        deniedLibraryPermissionLabel.textAlignment = .center
+        deniedLibraryPermissionLabel.text = "Live-Snap needs access to your photo library to import and export images. Please click the settings button below and enable access"
```

Comment:
avoid using the app name in strings as that might change later and we might forget to clean this up. either rephrase or make a constant variable in System that is the app name so we can use that one constant everywhere